lib/src/analyzers/lint_analyzer/rules/rules_list/avoid_collection_methods_with_unrelated_types/avoid_collection_methods_with_unrelated_types_rule.dart

```diff
@@ -3,6 +3,7 @@
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/nullability_suffix.dart';
```

Comment:
Is it imported but not used?

Reply:
That's how `part` and `part of` works - imports are listed only in "main" file. This one is used for `NullabilitySuffix.question`

Reply:
Oh I see. Only look at github review panel so not realized that